packages/core-design-system/README.md

```diff
@@ -0,0 +1,60 @@
+# Harness Design System
+
+This package provides the core design system for Harness UI components.
+
+## Installation
+
+```bash
+pnpm add @harnessio/core-design-system
+```
+
+## Usage
+
+### Importing CSS Styles
+
+To import all design system styles:
+
+```css
+@import '@harnessio/core-design-system/core-styles';
+@import '@harnessio/core-design-system/oss-styles';
```

Comment:
this should be `oss-theme` instead of `oss-styles` maybe?

Reply:
Updated